inc/cavalcade_runner_to_cloudwatch/namespace.php

```diff
@@ -1,19 +1,23 @@
 <?php
 
-namespace HM\Cavalcade\CloudWatch;
+namespace Altis\Cloud\Cavalcade_Runner_To_CloudWatch;
 
 use Aws\CloudWatch\CloudWatchClient;
 use Exception;
-use HM\Cavalcade\Runner\Hooks;
+use function Altis\get_aws_sdk;
 use HM\Cavalcade\Runner\Job;
 use HM\Cavalcade\Runner\Runner;
 use HM\Cavalcade\Runner\Worker;
 use ReflectionClass;
-use function Altis\get_aws_sdk;
 
-Runner::instance()->hooks->register( 'Runner.run_job.started', __NAMESPACE__ . '\\on_job_started' );
-Runner::instance()->hooks->register( 'Runner.check_workers.job_failed', __NAMESPACE__ . '\\on_job_failed' );
-Runner::instance()->hooks->register( 'Runner.check_workers.job_completed', __NAMESPACE__ . '\\on_job_completed' );
+/**
+ * Register Cavalcade Hooks.
+ */
+function bootstrap() {
```

Comment:
This will still run in the Cavalcade Runner context right?

Reply:
Yep, I added the call to it after the file is included here https://github.com/humanmade/altis-cloud/pull/45/files#diff-2d9bc91244a2cb57445498ad1c971f7dR48